docs/networks/arbitrum-sepolia.md

```diff
@@ -62,16 +66,25 @@ option can be used.
-| --------------------------------------------- | ------------------------------- | -------------------------------------------------------------------------------------- |
-| `INDEXER_SERVICE_ETHEREUM`                    | `--ethereum`                    | An Arbitrum Sepolia node/provider                                                      |
-| `INDEXER_SERVICE_INDEXER_ADDRESS`             | `--indexer-address`             | Ethereum address of testnet indexer                                                    |
-| `INDEXER_SERVICE_MNEMONIC`                    | `--mnemonic`                    | Ethereum mnemonic for testnet operator                                                 |
-| `INDEXER_SERVICE_NETWORK_SUBGRAPH_DEPLOYMENT` | `--network-subgraph-deployment` | `QmXnGVrg6DvscnvJd86aHAPLGyGrkM17weMrAsFAEMmQLL`                                       |
-| `INDEXER_SERVICE_NETWORK_SUBGRAPH_ENDPOINT`   | `--network-subgraph-endpoint`   | `https://api.thegraph.com/subgraphs/name/graphprotocol/graph-network-arbitrum-sepolia` |
-| `INDEXER_SERVICE_CLIENT_SIGNER_ADDRESS`       | `--client-signer-address`       | `0xac01B0b3B2Dc5D8E0D484c02c4d077C15C96a7b4`                                           |
+### Indexer Service rs *and* TAP Agent
+
+They are configured using a TOML file provided with `--config`. You should start with [the minimal config example](https://github.com/graphprotocol/indexer-rs/blob/main/config/minimal-config-example.toml). You can find the full config [here](https://github.com/graphprotocol/indexer-rs/blob/main/config/maximal-config-example.toml) and the default values 
+[here](https://github.com/graphprotocol/indexer-rs/blob/main/config/default_values.toml).
+
+| Variable                          | Value                                                                                                                   |
+| --------------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `indexer.indexer_address`         | Ethereum address of testnet indexer                                                                                     |
+| `indexer.operator_mnemonic`       | Ethereum mnemonic for testnet operator                                                                                  |
+| `subgraphs.network.deployment_id` | `QmXnGVrg6DvscnvJd86aHAPLGyGrkM17weMrAsFAEMmQLL`                                                                        |
+| `subgraphs.network.query_url`     | `https://gateway-arbitrum.network.thegraph.com/api/[api-key]/subgraphs/id/3xQHhMudr1oh69ut36G2mbzpYmYxwqCeU6wwqyCDCnqV` |
+| `subgraphs.escrow.deployment_id`  | `QmUiLdbsk6c51UMdcNBxsP3KadJpkmp6a3k2NCprR4ZFeM`                                                                        |
```

Comment:
@aasseman is it still the case that user is expected to provide either query_url OR deployment_id for subgraphs.networkand subgraphs.escrow? If it is we may want to make that clear here as well

Reply:
Ideally you supply both. But you must at least supply a `query_url`.

Reply:
BTW all the details about each configuration option is here: https://github.com/graphprotocol/indexer-rs/blob/main/config/maximal-config-example.toml
The link had been added here: https://github.com/graphprotocol/indexer/pull/1001/files#diff-9f5fd491a13bfb98a7bafabd8a2b358c37ba98dc15543b642b062b0c31ba16f6R71